docs/sections/getting_started.md

```diff
@@ -128,17 +128,25 @@ popper run
 
 You should see the output of actions printed to the terminal.
 
+To obtain more detailed information of what this command does, you can pass the `--help` flag to it:
+
+```bash
+popper run --help
+```
+
+> **Note**: all Popper subcommands allow you to pass `--help` flag to it to get more information about what the command does.
```

Comment:
can we change `Note` to `NOTE` to keep it consistent with the other parts of this page?

Reply:
Let me get right on this and close this PR